cumulus_library/studies/core/core_templates/documentreference.sql.jinja

```diff
@@ -0,0 +1,64 @@
+{% import 'core_utils.jinja' as utils %}
+CREATE TABLE core__documentreference AS
+WITH temp_documentreference AS (
+    SELECT DISTINCT
+        {{- utils.basic_cols(
+                'documentreference',
+                'dr',
+                [
+                    'id',
+                ]
+            ) 
+        }},
+        {{- utils.nullable_cols(
+                'documentreference',
+                'dr',
+                [
+                    'type',
+                    'status',
+                    'docstatus',
+                    'context',                
```

Comment:
super nit: trailing whitespace here that I happened to note because of a nervous highlighting tic :smile:

Reply:
part of the sqlfluff pass - though i should check my local settings, i would have expected this to be caught pre-commit.